akka-projection-dynamodb-integration/src/test/scala/akka/projection/dynamodb/TestConfig.scala

```diff
@@ -15,6 +15,7 @@ object TestConfig {
       .parseString("""
       akka.loglevel = DEBUG
       akka.persistence.journal.plugin = "akka.persistence.dynamodb.journal"
+      akka.persistence.snapshot-store.plugin = "akka.persistence.dynamodb.snapshot"
```

Comment:
Having the snapshot store enabled is causing some other tests to fail. We don't have this enabled for r2dbc either. I'll just remove it again for now. And can be enabled specifically for the end-to-end spec.

The end-to-end spec is also failing for me though.

Reply:
Ok, it's that we're not creating the snapshot table for tests. Will fix.

Reply:
Green now.